Disable the Prometheus exporter by default (#131)
mesosphere/kudo-cassandra-operator: docs/monitoring.md

achieves this using a Prometheus exporter based on the
[criteo/cassandra_exporter](https://github.com/criteo/cassandra_exporter).

> When the `PROMETHEUS_EXPORTER_ENABLED` parameter is set to `true`:

Reviewer: Please correct line 7 as well as 44/45.

### 1. Make sure that Prometheus Exporter is enabled on the KUDO Cassandra instance

> This parameter is `false` by default, so you only need to worry about this if
> you already have a KUDO Cassandra instance running.

Reviewer: ```suggestion
This parameter is `false` by default, so you need to enable it explicitly.
```

> If you do not remember, you can check the value of the parameter on a running

Reviewer: ```suggestion
You can check the value of the parameter on a running
```
